usdoj-crt/beta-ada
# Added in Jekyll redirect plugin

Changes:

1. Added the [Jekyll Redirect From](https://github.com/jekyll/jekyll-redirect-from) plugin
2. Added the appropriate fields to the Web Guidance and AI Guidance pages to handle redirects.

To test: 
1. Navigate to the old url for Web Guidance, which is preview-url/web-guidance. Confirm that you are redirected to preview-url/resources/web-guidance
2. Navigate to the old url for AI Guidance, which is preview-url/ai-guidance. Confirm that you are redirected to preview-url/resources/ai-guidance

If this works, it should close: https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/491
